lib/models/src/main/java/org/sagebionetworks/repo/model/AuthorizationConstants.java

```diff
@@ -200,6 +200,11 @@ public static boolean isBootstrapPrincipalId(long id) {
 	 */
 	public static final String SYNAPSE_OAUTH_CLIENT_ID = "0";
 	
+	/**
+	 * The time, in seconds, that an access token lasts
+	 */
+	public static final long ACCESS_TOKEN_EXPIRATION_TIME_SECONDS = 3600*24L; // a day
```

Comment:
Please double check with Aaron about the recommended expiration timeout for this kind of access token (if any).

Reply:
In progress (via Jira).